README.md

```diff
@@ -1,57 +1,147 @@
-# Project Name
+# [Spring PetClinic Sample Application](https://github.com/spring-projects/spring-petclinic) using a full set of Azure solutions
 
-(short, 1-3 sentenced, description of the project)
+* [Azure AppService](https://azure.microsoft.com/en-us/products/app-service/) for app hosting,
+* [Azure Database for MySQL](https://azure.microsoft.com/en-us/products/mysql/) for storage,
+* [Azure Monitor](https://azure.microsoft.com/en-us/products/monitor/)([Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview?tabs=net)) for monitoring and logging.
+* [Key Vault](https://azure.microsoft.com/en-us/products/key-vault/) for securing secret/password.
 
-## Features
+This application uses the [Azure Developer CLI (azd)](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/overview)
+to get you up and running on Azure quickly.
 
-This project framework provides the following features:
+## Diff from the original Spring PetClinic 
+diff from [9ecdc111](https://github.com/spring-projects/spring-petclinic/commit/9ecdc1111e3da388a750ace41a125287d9620534)
 
-* Feature 1
-* Feature 2
-* ...
+* `./infra` and `./azure.yaml`: application "infrastructure as code" files to create and configure Azure resources.
+* `./pom.xml` and `./build.gradle`: import required libraries for `Azure Key Vault`/`Azure Monitor`(`Azure Application Insights`) integration.
+    * `pom.xml`
+      ![diff of pom.xml](readme.assests/pom.xml.diff.png)
+    * `build.gradle`
+      ![diff of build.gradle](readme.assests/build.gradle.diff.png)
+* `./src/main/resources/application-azure.properties`: enable importing KeyVault secrets(`MySQL` user password) as properties.
+  see [Configurable properties of Key Vault Secret PropertySource](https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html#all-configurable-properties)
+  ```properties
+    # Import KeyVault secrets as properties
+    spring.cloud.azure.keyvault.secret.property-sources[0].enabled=true
+    spring.cloud.azure.keyvault.secret.property-sources[0].endpoint=${AZURE_KEY_VAULT_ENDPOINT}
+  ```
+* `./src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java`: enable monitoring and logging with `Azure Application Insights`
+  ![diff of PetClinicApplication.java](readme.assests/PetClinicApplication.java.diff.png)
 
-## Getting Started
+## Prerequisites
 
-### Prerequisites
+The following prerequisites are required to use this application. Please ensure that you have them all installed locally.
 
-(ideally very short, if any)
+* [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/install-azd)
+* Java 8 or later
```

Comment:
`azd` currently requires Java 17 SDK and above. I think it's generally a good idea for templates to be up-to-date, so I think it makes sense to bump this up here.

There's a question of whether that's too limiting from an `azd` perspective, but given that we're targeting newer audience, I think the strong requirement is fine for now. But let me know if you think otherwise.

Reply:
I'd probably think in terms of the minimum version recommended by Azure App Service/Azure Functions/Azure Spring Apps.
But as a user, I can understand if it requires java17 or above,